java/src/jmri/Block.java

```diff
@@ -680,9 +680,6 @@ public void goingActive() {
                 count++;
                 next = pList[i];
             }
-            log.debug("\n{} is {}\n{} is {}",
-                    Path.decodeDirection(pDir[i]), String.format("%16s", Integer.toBinaryString(pDir[i])).replace(' ', '0'),
```

Comment:
Thanks, I've done that elsewhere, but this one really was intended for my understanding of the code and very unlikely to be helpful later.